### ISSUE TITLE
Implement workaround for https://github.com/helm/helm/issues/3742

### ISSUE THREAD

**Issue by sudermanjr** [kubecost/cost-analyzer-helm-chart#74]
We use Reckoner to manage our infrastructure charts, and this particular issue blocks us being able to use it for installing kubecost.  There is a workaround for the issue [here](https://github.com/helm/helm/issues/3742#issuecomment-383095917) that would be nice to have.

I completely understand if you do not want to implement the workaround, but I thought I would request.

**Comment by AjayTripathy** [comment]
Based on that open issue, it looks like a fix is in flight already with open pull requests? I'd prefer to wait on those to land before doing a workaround.

**Comment by sudermanjr** [comment]
Seeing as that PR has been open for over a year, there doesn't seem to be a whole lot of hope in that department.  We have implemented workarounds in all of our charts due to the slow-moving nature of this particular bugfix. 

Of course, up to you to decide, and if you do decide to wait, feel free to close this.

**Comment by dwbrown2** [comment]
@sudermanjr this seems to be low priority for your team. Is that accurate? Happy to discuss further if it's a priority. Please feel free to reopen or reach out to team@kubecost.com!
